### DISCUSSION TITLE
Git Commit "More"

### DISCUSSION THREAD

**Discussion by tim-elmer** [requarks/wiki#6151] [Help / Questions] [answered]
Unless I completely missed a checkbox somewhere, the Git storage target doesn't include _all_ metadata; notably, any custom scripts and styles are omitted. Would it be possible to add this content to that synchronized by the Git storage target?

The most reasonable way to do this may be to generate/consume an additional JS/CSS file for each page if custom JS/CSS exists on the wiki or in the storage system depending on sync direction.

**Comment by NGPixel** [comment, marked as answer]
Submit your idea on https://requarks.canny.io/wiki
